packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -744,15 +690,36 @@ class _PathFormatCheck extends StatelessWidget {
   }
 }
 
-class _PathCheckExpansionTile extends StatelessWidget {
-  const _PathCheckExpansionTile({
+class _CheckTableHeader extends StatelessWidget {
+  const _CheckTableHeader();
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTile(
+      tileColor: Theme.of(context).colorScheme.deeplinkTableHeaderColor,
+      title: const Row(
+        children: [
+          SizedBox(width: defaultSpacing),
```

Comment:
instead of putting this in the row, how about putting a Padding widget around Row? this would make the intent of the spacer more clear